lib/suspenders/generators/accessibility_generator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative "base"
+
+module Suspenders
+  class AccessibilityGenerator < Generators::Base
```

Comment:
@seanpdoyle Should we hook up this generator to `app_generator.rb`? https://github.com/thoughtbot/suspenders/blob/b9d702007f544ada5f26a02d63e989bc6815b40f/lib/suspenders/generators/app_generator.rb#L147 ?

Reply:
Sure! I think that's a good idea. Where could we add test coverage for that behavior?

Reply:
This should probably be tested at the system [here](https://github.com/thoughtbot/suspenders/blob/b9d702007f544ada5f26a02d63e989bc6815b40f/spec/features/new_project_spec.rb). This spec file works with a `before(:all)` hook, so it's not that expensive to add a new example.